<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseWidget</class>
 <widget class="QWidget" name="DatabaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>369</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QGridLayout" name="database_grid">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="db_attributes_tbw">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <widget class="QWidget" name="attributes_tab">
      <attribute name="icon">
       <iconset resource="../../res/resources.qrc">
        <normaloff>:/icons/icons/attribute.png</normaloff>:/icons/icons/attribute.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Attributes</string>
      </attribute>
      <layout class="QGridLayout" name="settings_grid">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="connlimit_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="connlim_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Connection(s)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="connlim_sb">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="minimum">
            <number>-1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="lc_ctype_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lcctype_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>LC_CTYPE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="lcctype_cmb">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertPolicy::InsertAlphabetically</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="author_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="author_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Model author</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="author_edt">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QCheckBox" name="allow_conn_chk">
              <property name="text">
               <string>Allow connections</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="is_template_chk">
              <property name="text">
               <string>Is template</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="gen_sql_dis_objs_chk">
            <property name="text">
             <string>Generate SQL of disabled objects</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="lc_collate_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lccollate_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>LC_COLLATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="lccollate_cmb">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertPolicy::InsertAlphabetically</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="encoding_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="encoding_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Encoding</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="encoding_cmb">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="template_db_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="templatedb_lbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Template database</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="templatedb_edt">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="def_objects_tab">
      <attribute name="icon">
       <iconset resource="../../res/resources.qrc">
        <normaloff>:/icons/icons/objects.png</normaloff>:/icons/icons/objects.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Default objects</string>
      </attribute>
      <layout class="QGridLayout" name="def_objects_grid">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="0" column="0" colspan="4">
        <layout class="QVBoxLayout" name="def_collation_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="def_collation_lbl">
           <property name="text">
            <string>Collation</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0" colspan="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4">
        <layout class="QVBoxLayout" name="def_tablespace_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="def_tablespace_lbl">
           <property name="text">
            <string>Tablespace</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="4">
        <layout class="QVBoxLayout" name="def_schema_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="def_schema_lbl">
           <property name="text">
            <string>Schema</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="4">
        <layout class="QVBoxLayout" name="def_owner_lt">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="def_owner_lbl">
           <property name="text">
            <string>Owner</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
